app/gateway/traditional-mode.md

```diff
@@ -0,0 +1,158 @@
+`db_update_frequency` seconds later (though it could happen sooner).
+
+This makes {{site.base_gateway}} clusters **eventually consistent**.
+
+## Using read-only replicas when deploying {{site.base_gateway}} clusters with PostgresSQL
+
+When using Postgres as the backend storage, you can optionally enable
+{{site.base_gateway}} to serve read queries from a separate database instance.
+
+Enabling the read-only connection support in {{site.base_gateway}}
+greatly reduces the load on the main database instance since read-only
+queries are no longer sent to it.
+
+To learn more about how to configure this feature, refer to the
+[Datastore section](/gateway/configuration/#datastore-section)
+of the {{site.base_gateway}} configuration reference.
+
+## What information is cached?
+
+For performance reasons, {{site.base_gateway}} avoids database connections when proxying
+requests, and caches the contents of your database in memory. The following entities are cached:
+
+* [Gateway Services](/gateway/entities/service/)
+* [Routes](/gateway/entities/route/)
+* [Consumers](/gateway/entities/consumer/)
+* [Plugins](/gateway/entities/plugin/)
+* Credentials, and so on 
+
+Since these values are in memory, any change made via the [Admin API](/api/gateway/admin-ee/#/operations/) of one of the nodes must be propagated to the other nodes.
+
+Additionally, {{site.base_gateway}} also caches **database misses**. This means that if you
+configure a Service with no plugin, {{site.base_gateway}} will cache this information. 
+
+All CRUD operations trigger cache invalidations. Creation
+(`POST`, `PUT`) will invalidate cached database misses, and update/deletion
+(`PATCH`, `DELETE`) will invalidate cached database hits.
+
+## Configure database caching
+
+You can configure three properties in the {{site.base_gateway}} configuration file, the most
+important one being `db_update_frequency`, which determine where your {{site.base_gateway}}
+nodes stand on the performance versus consistency trade-off.
+
+{{site.base_gateway}} comes with default values tuned for consistency so that you can
+experiment with its clustering capabilities while avoiding surprises. As you
+prepare a production setup, you should consider tuning those values to ensure
+that your performance constraints are respected.
+
+<!--vale off-->
+{% kong_config_table %}
+config:
+  - name: db_update_frequency
+  - name: db_update_propagation
+  - name: db_cache_ttl
+{% endkong_config_table %}
+<!--vale on-->
+
+## Interacting with the cache via the Admin API
+
+If you want to investigate the cached values, or manually
+invalidate a value cached by {{site.base_gateway}} (a cached hit or miss), you can do so via the
+Admin API `/cache` endpoint.
+
+{:.info}
+> **Note**: Retrieving the `cache_key` for each entity being cached by {{site.base_gateway}} is
+currently an undocumented process. Future versions of the Admin API will make
+this process easier.
```

Comment:
This may be true, but it is odd. We should ask the gateway team how to find a cache key, I don't think we should have a note in the docs that say "we haven't documented this yet".
```suggestion
```

Reply:
Sounds good! I felt like it was odd too. Send a question about it in ask-gateway